app/assets/stylesheets/admin/_colors.scss

```diff
@@ -34,3 +34,5 @@ $barelyLate: hsl(56, 100%, 66%);
 $superLate: hsl(0, 74%, 54%);
 
 $veryLate: $orange2;
+
+$projectHoverColor: rgb(255, 255, 221);
```

Comment:
I would just call this `$highlight` or `$highlightColor`. The fact that it's used on hover in this case is incidental. We don't seem to be putting `Color` on the end of these things either...